docs/glossary.md

```diff
@@ -184,6 +221,19 @@ A group of miners who share (pool) their computational resources to mine DCR. Wh
 
 Tickets that have been called but did not receive a reward. This can happen if a ticket is called to vote, but the wallet that bought the ticket does not respond. This can also happen if the wallet does respond and broadcasts its vote to the network, but a miner does not include their vote in the following block. 
 
+#### Multisignature
+
+Multisignature refers to Decred addresses which require more than one private key to authorize transactions. A multisignature address can support multiple keys (N) and a subset of those (M) are required to transact (commonly known as "MofN"). For example, an 2 of 3 address would have three valid keys, however only two of them would be required to authorize transactions. 
```

Comment:
Mutlisignature does not refer to addresses, rather it refers to the conditions required to spend funds.

Reply:
Is this more accurate?

> Multisignature refers to transactions which can be authorized by more than one private key. Multisignature can support multiple keys (N) and a subset of those (M) are required to transact (commonly known as "MofN"). For example, an 2 of 3 multisignature would have three valid keys, however only two of them would be required to authorize transactions. 

I may not have a deep enough understanding of this to produce an accurate definition without an existing definition to work from.

Reply:
I think just adding the word `transactions` after multisignature in the latter two cases would do the trick.  It would be accurate then.

Reply:
Done. Thanks